pkg/printers/interface.go

```diff
@@ -94,3 +94,12 @@ type ErrNoDescriber struct {
 func (e ErrNoDescriber) Error() string {
 	return fmt.Sprintf("no describer has been defined for %v", e.Types)
 }
+
+// OutputOptions represents resource output options which is used to generate a resource printer.
+type OutputOptions struct {
+	FmtType string // e.g. json, yaml, template, jsonpath etc.
```

Comment:
Maybe this comment should point where I can find the actual list of possible values.